### ISSUE TITLE
bug: issue with 2.0.0b1 action

### ISSUE THREAD

**Issue by henryiii** [pypa/cibuildwheel#752]
Something is wrong with 2.0.0b1. Not sure what, debugging here. https://github.com/scikit-hep/boost-histogram/pull/605 Will update when I know more.

**Comment by henryiii** [comment]
It might be only affecting the action. The direct usage of pipx is working. The only thing I can think of is the `2>&1` addition, but that is [clearly supported](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_redirection?view=powershell-7.1), so don't know why that would be a problem.

**Comment by henryiii** [comment]
```yaml
    - run: pipx run --spec cibuildwheel==2.0.0b1 cibuildwheel 2>&1
      shell: pwsh
```

Breaks in a really weird way (exactly the same as the action). It gets through the first 30 lines or so, then just dies with no more output and fails on all three OS's. Have no idea why.

**Comment by henryiii** [comment]
I'm trying bash again just to see if they fixed the path format. Bash does handle the redirection correct AFAICT. Last time I tried it it was a Windows style path which didn't work in bash for this command, but did on powershell.

**Comment by henryiii** [comment]
Ahh, I think I see how to fix the path problem on bash. Let's try that.

**Comment by joerick** [comment]
I think this is resolved by v2.0.0b2?

**Comment by henryiii** [comment]
Yes, it is! https://github.com/scikit-hep/boost-histogram/pull/605 is merged now that I modified the bump to b2. Actually rather impressed that dependabot seems to understand it's okay to bump a dev release to another dev release.
